README.md

```diff
@@ -52,6 +53,34 @@ There are many (too many!) ways to write React components so to make sure we all
 |       ├── component-name.scss
 ```
 
+## Working with repositories
```

Comment:
can we move this to different file => `docs/gitflow` so this readme it;s not too long